gsoc/2020/posts/jose_miguel_sanchez/post01.md

```diff
@@ -1,15 +1,39 @@
-# GSoC'20 RoboComp project: NAME OF THE PROJECT
+# GSoC'20 RoboComp project: Syntax-error highlighter for LearnBlock
  
 DATE
  
 ## About me
  
-SOME INFORMATION ABOUT YOU
-
+Hello! I'm José Miguel, a Computer Engineering student in Cáceres, Spain.
+I've grown up with a passion for technology, and over time I've played around
+with computer graphics, programming language and operating system design,
+game modding, web design and programming, a bit of computer security...
+I love learning something from every field I discover!
 
 ## About the Project
 
-DESCRIPTION OF THE PROJECT AND ITS GOALS
+LearnBlock is a visual robot programming tool which eases the path towards
+learning full-fledged robot programming (in Python) by putting some simpler
+layers in between to make it easier for newcomers to implement their ideas and
+get results quickly. I was amazed when I discovered it, not only you have a
+rich visual block environment to code (and a simplified text language in the
+middle), but the resulting code is portable among a variety of robots!
+
+My goal with this project is to fix one critical flaw it has right now: lack of
```

Comment:
Te pongo todo el parrafo, pq me resutla más sencillo. Es lo mismo que tu quieres decir un poco más ordenador y tratando de corregir algunos pequeños mistakes. Pero vamos que está muy bien la prosa tb!

My goal with this project is to fix one critical flaw it has right now: lack of proper error reporting. In learning tools, it is crucial to have a useful guide for the resolution of errors, since the target user usually lacks the intuition or experience that an experienced developer has. The environment should not only point to the place it found a problem, but provide some tips which make the solution clear and explain why it is required. Then, the user has not only has solved their problem but has also learned why the previous answer was incorrect and how to fix it in the future.